Sources/PostgresNIO/Connection/PostgresConnection+CopyFrom.swift

```diff
@@ -0,0 +1,230 @@
+/// Handle to send data for a `COPY ... FROM STDIN` query to the backend.
+public struct PostgresCopyFromWriter: Sendable {
+    private let channelHandler: NIOLoopBound<PostgresChannelHandler>
+    private let eventLoop: any EventLoop
+
+    init(handler: PostgresChannelHandler, eventLoop: any EventLoop) {
+        self.channelHandler = NIOLoopBound(handler, eventLoop: eventLoop)
+        self.eventLoop = eventLoop
+    }
+
+    private func writeAssumingInEventLoop(_ byteBuffer: ByteBuffer, _ continuation: CheckedContinuation<Void, any Error>) {
+        precondition(eventLoop.inEventLoop)
+        let promise = eventLoop.makePromise(of: Void.self)
+        self.channelHandler.value.checkBackendCanReceiveCopyData(promise: promise)
+        promise.futureResult.flatMap {
+            if eventLoop.inEventLoop {
+                return eventLoop.makeCompletedFuture(withResultOf: {
+                    try self.channelHandler.value.sendCopyData(byteBuffer)
+                })
+            } else {
+                let promise = eventLoop.makePromise(of: Void.self)
+                eventLoop.execute {
+                    promise.completeWith(Result(catching: { try self.channelHandler.value.sendCopyData(byteBuffer) }))
+                }
+                return promise.futureResult
+            }
+        }.whenComplete { result in
+            continuation.resume(with: result)
+        }
+    }
+
+    #if compiler(>=6.0)
+    /// Send data for a `COPY ... FROM STDIN` operation to the backend.
+    ///
+    /// - Throws: If an error occurs during the write of if the backend sent an `ErrorResponse` during the copy 
+    ///   operation, eg. to indicate that a **previous** `write` call had an invalid format.
+    public func write(_ byteBuffer: ByteBuffer, isolation: isolated (any Actor)? = #isolation) async throws {
```

Comment:
lets remove the `isolation` calls here and instead adopt `nonisolated nonsending` for the whole package.
```suggestion
    public func write(_ byteBuffer: ByteBuffer) async throws {
```